src/device/intrinsics/math.jl

```diff
@@ -298,6 +310,7 @@ using SpecialFunctions
 
 @device_override Base.fma(x::Float64, y::Float64, z::Float64) = ccall("extern __nv_fma", llvmcall, Cdouble, (Cdouble, Cdouble, Cdouble), x, y, z)
 @device_override Base.fma(x::Float32, y::Float32, z::Float32) = ccall("extern __nv_fmaf", llvmcall, Cfloat, (Cfloat, Cfloat, Cfloat), x, y, z)
+# @device_override Base.fma(x::Float16, y::Float16, z::Float16) = @asmcall("fma.rnd.f16 \$0, \$1, \$2, \$3", "=h,h,h,h", Float16, Tuple{Float16, Float16, Float16}, x, y, z)
```

Comment:
Why disabled?

Reply:
I've re-enabled it for now, and it seems to work fine. But it looks like there's no corresponding test case in `device/intrinsics`

Reply:
May be a good time to add one :-)

Reply:
Added. See below.